app/_how-tos/configure-scim-for-insomnia-with-azure.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Configure SCIM for Insomnia with Azure
+description: Learn how to set up SCIM to manage Insomnia users and teams with Azure.
+content_type: how_to
+automated_tests: false
+products:
+    - insomnia
+
+tier: enterprise
+
+tags:
+  - sso
+  - azure
+  - authentication
+  - security
+
+tldr:
+    q: How do I configure SCIM with Azure in Insomnia?
+    a: Enable SCIM in Insomnia to generate a connection URL and token, then configure your Azure SAML application to enable SCIM provisioning and use the URL and token generated to integrate with Insomnia.
+
+related_resources:
+  - text: SCIM for Insomnia
+    url: /insomnia/scim/
+  - text: SSO for Insomnia
+    url: /insomnia/sso/
+  - text: Configure Azure SAML SSO in Insomnia
+    url: /how-to/azure-saml-sso-insomnia/
+
+prereqs:
+  inline:
+  - title: Insomnia role
+    include_content: prereqs/insomnia-owner
+    icon_url: /assets/icons/insomnia/insomnia.svg
+  - title: Verified domain
+    include_content: prereqs/insomnia-verified-domain
+    icon_url: /assets/icons/domain.svg
+  - title: Azure permissions
+    include_content: prereqs/azure-sso
+    icon_url: /assets/icons/azure.svg
+  - title: Azure SSO
+    include_content: prereqs/insomnia-sso-azure
+    icon_url: /assets/icons/azure.svg
+---
+
+## Enable SCIM in Insomnia
+
+{% include /how-tos/steps/insomnia-scim.md %}
+
+## Configure SCIM in Azure
+
+1. In your [Azure app](#azure-sso), go to **Manage** > **Provisioning** and set **Provisioning Mode** to **Automatic**.
```

Comment:
```suggestion
1. In the [Azure app you created](#azure-sso) in the prerequisites, go to **Manage** > **Provisioning** and set **Provisioning Mode** to **Automatic**.
```
I think that edit is correct, we're trying to get them to go to the thing they made in the prereqs?